.github/workflows/release.yml

```diff
@@ -102,14 +102,14 @@ jobs:
       matrix:
         include: "${{ fromJSON(needs.metadata.outputs.matrix)['build-packages'] }}"
 
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-22.04
     container:
       image: ${{ matrix.image }}
       options: --privileged
 
     steps:
     - name: Early Rpm Setup
-      if: matrix.package == 'rpm'
+      if: matrix.package == 'rpm' && matrix.image != ''
```

Comment:
From line 105, it seems run only on Ubuntu. Then is this `matrix.package == 'rpm'` check valid?

Reply:
@zhongweiy currently, we run following builds a on bare ubuntu runner (VM):
- ubuntu-22.04 builds x86 + arm64 (crossbuild)
- rhel9, aws2023 arm64 (crossbuild) <= this PR was doing this
the following are ran in a container inside a ubuntu runner VM:
- all other rpm platforms in x86
- debian x86 + arm64
- ubuntu 20.04 x86

Reply:
Got it, thanks for the explanation!